### PR TITLE
[AV2-601] WCAG color requirements

### PR DESCRIPTION
**Pull request by sasangachathumal**
Jira - https://intersective.atlassian.net/browse/AV2-601
In this story only fixing font colors. not font sizes.

### CODE REVIEW

**Comment by sasangachathumal** [comment]
PR still not ready for code review

File: src/theme/pfas/_colors.scss [hunk 1]
```diff
@@ -5,16 +5,24 @@
...
 .gray-3 {
-  color: #565B70 !important;
+  color: rgba(0, 0, 0, 0.8) !important;
+}
+.black {
+  color: black !important;
+}
+// use in activity, event components
+// for list item shared component for expired event title
+.color-gray-2 {
```
Comment (shawnm0705) [inline src/theme/pfas/_colors.scss]: no need anymore

Comment (sasangachathumal) [reply src/theme/pfas/_colors.scss]: @shawnm0705 In activity component and in events we pass this 
`[titleColor]="eventListService.isNotActionable(event) ? 'gray-2' : ''"`
and in list item component we put title color like this
`<p class="item-title subtitle-1" [ngClass]="'color-' + titleColor">{{ title }}</p>`

Comment (sasangachathumal) [reply src/theme/pfas/_colors.scss]: that's why i create a class color-gray-2

Comment (shawnm0705) [reply src/theme/pfas/_colors.scss]: Then we just need to put this class inside the css of list-item component only, it is not used anywhere else

**Pull request merged**